features/step_definitions/authentication.js

```
@@ -21,13 +21,18 @@ var apiSteps = function () {
           this.currentUser = {};
         }
 
-        this.currentToken = { jwt: body.result.jwt };
+        this.currentToken = {jwt: body.result.jwt};
         this.currentUser.token = body.result.jwt;
 
         callback();
```

should throw an error if `cantLogin` is `true`